python/sidecar/interact.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+import sys
+
+import openai
+from dotenv import load_dotenv
+from sidecar import prompts
+
+load_dotenv()
+openai.api_key = os.environ.get("OPENAI_API_KEY")
```

Comment:
@gjreda I've changed this from index notation to `.get(...)` to fix the tests.

I'm not sure if this is the best approach but I feed that we shouldn't depend on this ENV var in the tests.

Reply:
I've been thinking about how to test our code when we depend on OpenAI APIs myself, so this PR is a good reminder to open an issue: https://github.com/refstudio/refstudio/issues/88. Let's put thoughts there?

Reply:
@cguedes I think either works. We are not using the API key at all in tests, since we are [mocking the call and response](https://github.com/refstudio/refstudio/pull/66/files#diff-bf83c0d6ab5fabad80921af7c7ac063d3524bb464bb2db06e5fbe240916d2cf7R7-R31). The OpenAI API never actually gets called.

Without the `get`, if the env variable is missing, an exception would get raised once the package was loaded. With the `get`, an exception will get raised once we try to call the API and realize the key is missing. Either way, it shouldn't have an impact on the tests.

Reply:
Said differently, in testing, we could set the env variable to `test` and it shouldn't make a difference.